packages/shared/src/components/sidebar/InvitePeople.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react';
+import { useCopyLink } from '../../hooks/useCopyLink';
+import { ButtonOrLink, ItemInner, ListIcon, NavItem } from './common';
+import UserShareIcon from '../../../icons/user_share.svg';
+
+export default function InvitePeople({
+  openSidebar,
+}: {
+  openSidebar: boolean;
+}): ReactElement {
+  const [copyingLink, copyLink] = useCopyLink(() => 'https://daily.dev/');
```

Comment:
Please validate with Nimrod what should be the right link to share.
This can be either our landing page as you did now or `https://api.daily.dev/get` which is a direct link to the relevant store.